### PR TITLE
Update Get-AzureRmRoleAssignment -IncludeClassicAdministrators

### PR DESCRIPTION
**Pull request by sneivandt**
## Description

Update `Get-AzureRmRoleAssignment` to use the subscription scope if it is provided when requesting classic administrators.

This change is being made to address issue #7679.

This change makes a request the one bellow include classic admins for the subscription `$subscriptionId` . Currently behavior is to included classic admins for the subscription that is selected.

``` 
Get-AzureRmRoleAssignment -Scope '/subscriptions/$subscriptionId' -IncludeClassicAdministrators
```

### TODO

We require a second subscription where the test user is not a classic administrator to cover this scenario in our tests. I will update the Session Records once the new subscription is correctly configured. Testing I have already done was done against my personal subscription.

- [x] Update Session Records

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
@sneivandt Ping

**Comment by sneivandt** [comment]
@maddieclayton I am waiting for the second subscription I mentioned to be configured correctly. I will update this including the test recordings when that work is done.

File: src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.ps1 [hunk 1]
```diff
@@ -19,7 +19,7 @@ Tests retrieval of classic administrators
 function Test-RaClassicAdmins
 {
     # Setup
-    $subscription = Get-AzureRmSubscription
+    $subscription = Get-AzureRmSubscription | Where-Object { $_.Id -ieq "4004a9fd-d58e-48dc-aeb2-4a4aec58606f" }
```
Comment (cormacpayne) [inline src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.ps1]: @sneivandt do we need to include these specific subscriptions and accounts in all of the tests below? This makes it difficult for users who don't have access to these subscriptions and accounts to re-record these tests if they need to in the future.

CC: @darshanhs90

**Comment by cormacpayne** [comment]
@sneivandt @darshanhs90 ping on the above comment

**Comment by sneivandt** [comment]
@cormacpayne I have been out but I intend to make a revision to address your comment soon. It should be possible to remove the requirement to use exactly these subscriptions.

File: src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.ps1 (hunk 1 above)
Comment (sneivandt) [reply src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.ps1]: I made some changes and the current version no longer mentions specific subscriptions or accounts.

**Pull request merged**